core/src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala

```diff
@@ -496,6 +496,19 @@ trait DeltaConfigsBase extends DeltaLogging {
     opt => opt.forall(isValidIntervalConfigValue),
     "needs to be provided as a calendar interval such as '2 weeks'. Months " +
       "and years are not accepted. You may specify '365 days' for a year instead.")
+
+  /**
+   * The isolation level of a table defines the degree to which a transaction must be isolated from
+   * modifications made by concurrent transactions. Delta supports two isolation levels:
+   * Serializable and WriteSerializable. The default is Serializable.
+   */
+  val ISOLATION_LEVEL = buildConfig[IsolationLevel](
+    "isolationLevel",
+    Serializable.toString,
+    IsolationLevel.fromString(_),
+    _ => true,
+    "needs to be Serializable or WriteSerializable"
```

Comment:
Since we don't have the implementation for WriteSerializable yet, could you make this config only accept  Serializable?

We can target this PR to allow users to set `delta.isolationLevel` to `Serializable`. This is correct and there is no point to block people.

Supporting `WriteSerializable` can be a separate PR that would require a large change.

Reply:
I just took a quick look at the existing code for `WriteSerializable`. It might not be correct. I need to take a detail look and think through it. Haven't read these codes for a while. This is the most complicated part in Delta.

Meanwhile, you can change the code to enable Serializable only. Will open it to allow WriteSerializable when we fix the bugs.

Reply:
Created https://github.com/delta-io/delta/pull/1276 to fix https://github.com/delta-io/delta/issues/1265
This is just to allow setting Serializable isolation level only

Reply:
i did an experiment where i changed default isolation level to `WriteSerializable` and ran all the tests. the only tests that failed where the ones that should fail due to the change in behavior.
we also started testing multiple jobs writing concurrently to same table using `WriteSerializable` isolation level on real cluster and everything is behaving as expected.
i am in no way saying the code for `WriteSerializable` is correct, but if its not i haven't spotted the issues so far.
just FYI